src/views/forms/FormTransactionBase/FormTransactionBase.ts

```diff
@@ -107,6 +130,11 @@ export class FormTransactionBase extends Vue {
    */
   public currentSigner: PublicAccount
 
+  /**
+   * Currently active wallet
+   */
+  // public currentWallet: WalletModel
```

Comment:
`currentWallet` is used in subclasses of FormTransactionBase ; why remove it ?

Reply:
The same as the comment above.

Reply:
@fboucquez ? is this correct?